ekncontent/ekncontent/eknc-utils.c

```diff
@@ -198,6 +198,106 @@ eknc_utils_parallel_init (GSList        *initables,
     }
 }
 
+static GFile *
+database_dir_from_data_dir (const gchar *data_dir, const gchar *app_id)
+{
+  g_autofree gchar *database_dir = g_build_filename (data_dir, "ekn", "data",
+                                                     app_id, NULL);
+  GFile *file = g_file_new_for_path (database_dir);
+  if (g_file_query_exists (file, NULL))
+      return file;
+  g_object_unref (file);
+  return NULL;
+}
+
+/**
+ * eknc_get_running_under_flatpak
```

Comment:
no, which is odd. gtk-doc is definitely working in make check though, im using that regularly to get all my symbols properly in sections